From 65370045321e97ee1cc82ead3fd6bb7c4b493b90 Mon Sep 17 00:00:00 2001
From: Juan RP <xtraeme@gmail.com>
Date: Wed, 8 Jan 2014 20:16:02 +0100
Subject: [PATCH 3/3] xbps-create: do not use FTW_MOUNT with nftw(3).

There's not much point in using FTW_MOUNT anyway because we don't care
if files are in another filesystem. This makes it work with musl
(which uses a different behaviour when FTW_MOUNT is set).
---
 bin/xbps-create/main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bin/xbps-create/main.c b/bin/xbps-create/main.c
index 1624539..6223f1c 100644
--- a/bin/xbps-create/main.c
+++ b/bin/xbps-create/main.c
@@ -354,7 +354,7 @@ process_xentry(const char *key, const char *mutable_files)
 static void
 process_destdir(const char *mutable_files)
 {
-	if (nftw(".", ftw_cb, 20, FTW_PHYS|FTW_MOUNT) != 0)
+	if (nftw(".", ftw_cb, 20, FTW_PHYS) != 0)
 		die("failed to process destdir files (nftw):");
 
 	/* Process regular files */
-- 
1.8.5.2

